.github/workflows/prettier.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Format with Prettier'
+on:
+  pull_request:
+      branches: [master]
```

Comment:
Is there a reason this one only runs when PR'ing against master?

Reply:
No, I just copied an example

Reply:
Ah, then it should be removed (for when we rename the branch to `main` :wink:)

Reply:
This seems to be required. I do not know why